crates/bevy_reflect/bevy_reflect_derive/src/utility.rs

```diff
@@ -0,0 +1,6 @@
+use bevy_macro_utils::BevyManifest;
+use syn::Path;
+
+pub fn get_bevy_reflect_path() -> Path {
```

Comment:
This really feels like it belong in `bevy_macro_utils` as a const fn.

Reply:
I'm not sure it can be `const`. Looking deeper into `get_path` it seems pretty involved...

Reply:
Fair. I'm totally fine with it being non-const. I do still think it should be moved.

Reply:
Actually, to clarify on this, do you want there to be a `BevyManifest::get_path(...)` method or do you want there to specifically be a `BevyManifest:: get_bevy_reflect_path()` method?

Reply:
The former :)